docs/ops/sequence/CTCGreedyDecoderSeqLen_6.md

```diff
@@ -0,0 +1,90 @@
+## CTCGreedyDecoderSeqLen <a name="CTCGreedyDecoderSeqLen"></a> {#openvino_docs_ops_sequence_CTCGreedyDecoderSeqLen_6}
+
+**Versioned name**: *CTCGreedyDecoderSeqLen-6*
+
+**Category**: Sequence processing
+
+**Short description**: *CTCGreedyDecoderSeqLen* performs greedy decoding of the logits provided as the first input. The sequence lengths are provided as the second input.
+
+**Detailed description**:
+
+This operation is similar to TensorFlow CTCGreedyDecoder [Reference](https://www.tensorflow.org/api_docs/python/tf/nn/ctc_greedy_decoder)
+
+The operation *CTCGreedyDecoderSeqLen* implements best path decoding.
+Decoding is done in two steps:
+
+1. Concatenate most probable characters per time-step which yields the best path.
+
+2. Then, undo the encoding by first removing duplicate characters and then removing all blanks. This gives us the recognized text.
```

Comment:
I don't understand this statement. Should it be something like this:

"Remove duplicate consecutive elements if the attribute `merge_repeated` is true and then remove all blank elements."

Reply:
"This gives us the recognized text." - there is nothing in the spec says that the input is a text. And in fact it is not a text at all. We should use some neutral word to specify objects the operation works with.